src/main/java/com/google/edith/servlets/WeekInfo.java

```diff
@@ -0,0 +1,15 @@
+package com.google.edith.servlets;
+
+public class WeekInfo {
+    private String date;
```

Comment:
How is this string represented? I.e. what is the date convention that should be followed or is expected.
eg "YYYY-MM-DD" ? Do we need to add any checks to ensure the convention is followed ?

Reply:
It is "yyyy-mm-dd". There are no checks needed in this class because the class implementing this one will have already checked for the correct string representation.